frontend/src/components/commons/Notepad.jsx

```diff
@@ -1,102 +1,180 @@
-        this.setState({ columnWidth: 18 });
+      //When Opening the Notepad
+      case "handleOpen":
+        this.setState({ notepadHidden: false });
         break;
       default:
-        this.setState({ columnWidth: 18 });
+        this.setState({ notepadHidden: false });
     }
   };
 
-  componentDidMount() {
-    this.detectBrowser();
-  }
-
   render() {
+    const { notepadHidden } = this.state;
     return (
       <div style={styles.windowPadding}>
-        <div style={styles.content}>
-          <div style={styles.title}>
-            <h4 style={styles.h4}>{LOCALIZE.commons.notepad.title}</h4>
+        {!notepadHidden && (
+          <div style={styles.content}>
+            <div style={styles.hideNotepadBtnZone}>
+              <span
+                onClick={() => this.handleCollapse("handleHide")}
+                style={styles.hideNotepadBtnIcon}
+                className="fas fa-minus-circle"
+              />
+              <button
+                onClick={() => this.handleCollapse("handleHide")}
+                style={styles.hideNotepadBtn}
+              >
+                {LOCALIZE.commons.notepad.hideButton}
+              </button>
+            </div>
+            <div>
+              <h4 style={styles.h4}>{LOCALIZE.commons.notepad.title}</h4>
+            </div>
+            <div style={styles.notepadSection}>
+              <form id="notepad-content">
+                <fieldset>
+                  <label htmlFor="text-area-zone" className="invisible position-absolute">
+                    {LOCALIZE.commons.notepad.title}
+                  </label>
+                  <TextareaAutosize
+                    id="text-area-zone"
+                    maxLength="3000"
+                    className="text-area"
+                    style={styles.textArea}
+                    cols="45"
+                    minRows={5}
+                    placeholder={LOCALIZE.commons.notepad.placeholder}
+                    defaultValue={this.state.notepadContent}
+                  />
+                </fieldset>
+              </form>
+            </div>
           </div>
-          <div style={styles.notepadSection}>
-            <form>
-              <fieldset style={styles.center}>
-                <label htmlFor="text-area-zone" className="invisible position-absolute">
-                  {LOCALIZE.commons.notepad.title}
-                </label>
-                <TextareaAutosize
-                  id="text-area-zone"
-                  maxLength="3000"
-                  className="text-area"
-                  style={styles.textArea}
-                  cols={this.state.columnWidth}
-                  minRows={5}
-                  placeholder={LOCALIZE.commons.notepad.placeholder}
-                />
-              </fieldset>
-            </form>
+        )}
+        {notepadHidden && (
+          <div style={styles.openNotepadBtnHeight}>
+            <span
+              onClick={this.handleOpen}
```

Comment:
This doesn't use a parameter. Is this functional?

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/150/commits/f77628c93f8cdafaf39f7a331b9cfebfed366346 and commit #https://github.com/code-for-canada/project-thundercat/pull/150/commits/ef911b4c86bdd6dab57521372ff853cb20ee697b